Makefile

```diff
@@ -235,7 +235,7 @@ _lint-fmt: _pull-tf
 	@echo "# *.tf files"
 	@echo "------------------------------------------------------------"
 	@if docker run $$(tty -s && echo "-it" || echo) --rm -v "$(CURRENT_DIR):/t:ro" --workdir "/t" hashicorp/terraform:$(TF_VERSION) \
-		fmt -recursive -check=true -diff=true -write=false -list=true .; then \
+		fmt -recursive -check=true -diff=true -write=true -list=true .; then \
```

Comment:
FYI: All `lint` stages are currently read-only and do not change anything. Why the change here?

Reply:
copy & paste from here https://github.com/Flaconi/terraform-cloudflare-zone/blob/553805b3a99856eea8f62815bb50651ad4ead3eb/Makefile#L238